### PR TITLE
Use crates_source on stratisd 3.1.0

### PR DESCRIPTION
**Pull request by bgurney-rh**
Related: #265

### CODE REVIEW

File: blackbox/stratisd.spec [hunk 1]
```diff
@@ -40,9 +40,8 @@ Stratisd test build.  This package should not be used in production
 
 
 %prep
-%setup -q -n %{name}-%{version}
-%patch0 -p1
-
+%autosetup -p1
```
Comment (mulkieran) [inline blackbox/stratisd.spec:43]: Better off going back to ```%setup -q``` here. With this setup, we will have to apply any patches _after_ untarring the Sourc2 file, not before.

**Review by mulkieran — CHANGES_REQUESTED** [review]
Other than that request, if this works then I think that it is good. If it didn't work, then stratisd would fail to compile, so we would notice.

File: blackbox/stratisd.spec [hunk 2]
```diff
@@ -40,9 +40,8 @@ Stratisd test build.  This package should not be used in production
...
+tar xvf %{SOURCE2}
```
Comment (mulkieran) [inline blackbox/stratisd.spec:44]: I think '--strip-components=1" is necessary here.

**Review by mulkieran — CHANGES_REQUESTED** [review]

File: blackbox/stratisd.spec (hunk 1 above)
Comment (bgurney-rh) [reply blackbox/stratisd.spec]: Resolved.

File: blackbox/stratisd.spec (hunk 2 above)
Comment (bgurney-rh) [reply blackbox/stratisd.spec]: Resolved.

**Review by mulkieran — APPROVED** [review]

**Comment by mulkieran** [comment]
@bgurney-rh This looks good. Plz squash and I'll merge.

**Pull request merged**